packages/agreement-process/package.json

```diff
@@ -28,13 +28,13 @@
     "@types/uuid": "9.0.8",
     "cpx2": "2.0.0",
     "openapi-zod-client": "1.15.1",
+    "pagopa-interop-commons-test": "workspace:*",
     "pg-promise": "11.5.0",
     "prettier": "2.8.8",
     "testcontainers": "10.2.2",
     "ts-node": "10.9.2",
     "typescript": "5.1.3",
-    "pagopa-interop-commons-test": "workspace:*",
-    "vitest": "0.33.0"
+    "vitest": "1.5.0"
```

Comment:
The function used in order to pass data between setup functions "provide/inject" are available only on vitest versions `1.*`